src/main/java/serverutils/handlers/ServerUtilitiesPlayerEventHandler.java

```diff
@@ -77,15 +78,15 @@ public void onPlayerLoggedIn(ForgePlayerLoggedInEvent event) {
 
         Backups.hadPlayer = true;
 
-        if (ServerUtilitiesConfig.chat.replace_tab_names) {
-            new MessageUpdateTabName(player).sendToAll();
-
-            for (EntityPlayerMP player1 : player.mcServer.getConfigurationManager().playerEntityList) {
-                if (player1 != player) {
-                    new MessageUpdateTabName(player1).sendTo(player);
-                }
-            }
-        }
+        // if (ServerUtilitiesConfig.chat.replace_tab_names) {
+        // new MessageUpdateTabName(player).sendToAll();
+        //
+        // for (EntityPlayerMP player1 : player.mcServer.getConfigurationManager().playerEntityList) {
+        // if (player1 != player) {
+        // new MessageUpdateTabName(player1).sendTo(player);
+        // }
+        // }
+        // }
```

Comment:
Could delete this if its not needed

Reply:
Yeah you're right. I kept it as a bad reminder to fix it but I'll forget regardless...